build/configs/artik053/rssi_report/defconfig

```diff
@@ -217,9 +216,9 @@ CONFIG_BOOT_RUNFROMFLASH=y
 #
 # Boot Memory Configuration
 #
-CONFIG_RAM_REGIONx_START="0x02023800"
-CONFIG_RAM_REGIONx_SIZE="968704"
-CONFIG_RAM_KREGIONx_HEAP_INDEX="0,"
+CONFIG_RAM_KREGIONx_START="0x02023800"
+CONFIG_RAM_KREGIONx_SIZE="968704"
+CONFIG_RAM_MALLOC_PRIOR_INDEX=0
```

Comment:
In this defconfig, we only use the single heap. In this case, we don't need this config.

Reply:
This is not a part of this PR. Even though I entered menuconfig, this is not disappeared. To remove this, we should check Kconfig first. Let's do that with another PR.